src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const total = days * 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return total;
+  } else if (days < 7) {
+    return total - smallDiscount;
+  }
```

Comment:
else is redundant here
```suggestion
  }
  
  if (days < 7) {
    return total - smallDiscount;
  }
```